Source/Core/Core/Movie.cpp

```diff
@@ -156,19 +157,22 @@ std::string GetInputDisplay()
   if (!IsMovieActive())
   {
     s_controllers = 0;
-    for (int i = 0; i < 4; ++i)
+    for (int i = 0; i < SerialInterface::MAX_SI_CHANNELS; ++i)
     {
       if (SerialInterface::GetDeviceType(i) != SerialInterface::SIDEVICE_NONE)
         s_controllers |= (1 << i);
+    }
+    for (int i = 0; i < MAX_BBMOTES; ++i)
+    {
       if (g_wiimote_sources[i] != WIIMOTE_SRC_NONE)
-        s_controllers |= (1 << (i + 4));
+        s_controllers |= (1 << (i + SerialInterface::MAX_SI_CHANNELS));
     }
   }
 
   std::string input_display;
   {
     std::lock_guard<std::mutex> guard(s_input_display_lock);
-    for (int i = 0; i < 8; ++i)
+    for (int i = 0; i < SerialInterface::MAX_SI_CHANNELS + MAX_BBMOTES; ++i)
```

Comment:
s_InputDisplay.size() is more concise here (`i` will need to be changed to size_t, but this is fine)